tests/src/test/scala/cats/tests/OneAndSuite.scala

```diff
@@ -1,49 +1,22 @@
 package cats
 package tests
 
-import cats.kernel.laws.discipline.{EqTests, SemigroupTests}
```

Comment:
`NonEmptyStream` related tests in this suite were moved to `NonEmptyStreamSuite` which is now 2.12- specific